app/src/main/java/com/github/orkest/View/NavigationBar.kt

```diff
@@ -0,0 +1,69 @@
+package com.github.orkest.View
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.BottomNavigation
+import androidx.compose.material.BottomNavigationItem
+import androidx.compose.material.Icon
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavDestination.Companion.hierarchy
+import androidx.navigation.NavGraph.Companion.findStartDestination
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.currentBackStackEntryAsState
+
+class NavigationBar {
+
+    companion object{
+    @OptIn(ExperimentalMaterial3Api::class)
```

Comment:
An experimental API could be subject to changes in the future (breaking changes, bugs or other issues). Do the benefits of using this particular one overweight its downsides ?

Reply:
I understand the concerns, TBH i just used the one they used in all the compose tutorial "Build Jetpack Compose UIs with Material Design 3 Components, the next evolution of Material Design."